moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -716,7 +716,7 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
 
   MoveItErrorCode planGraspsAndPick(const moveit_msgs::CollisionObject& object, bool plan_only = false)
   {
-    if (!plan_grasps_service_)
+    if (!plan_grasps_service_.waitForExistence(ros::Duration(0.2)))
```

Comment:
How did you pick this duration for the wait?  Should it not just be 0 to make it fail right away if the service is not available?

Reply:
```suggestion
    if (!plan_grasps_service_.exists()) 
```